userprofile/management/commands/delete_users.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from course.models import CourseInstance
+from django.utils import timezone
+
+
+class Command(BaseCommand):
+    help = 'Delete inactive users based on the last_login timestamp, and course enrollment status'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-d', '--days-ago', type=int, default=365,
+            help='Count the number of users who have logged in in the past N days. '
```

Comment:
The help text is wrong. Copy-pasted from another file?

Reply:
fixed